controllers/humiocluster_defaults.go

```diff
@@ -197,6 +199,7 @@ func NewHumioNodeManagerFromHumioNodePool(hc *humiov1alpha1.HumioCluster, hnp *h
 			InitServiceAccountName:         hnp.InitServiceAccountName,
 			PodLabels:                      hnp.PodLabels,
 			UpdateStrategy:                 hnp.UpdateStrategy,
+			PriorityClassName:              hc.Spec.PriorityClassName,
```

Comment:
I think this should be:
```suggestion
			PriorityClassName:              hnp.PriorityClassName,
```